modules/virt-deploying-operator-cli.adoc

```diff
@@ -35,7 +46,12 @@ $ oc apply -f <file_name>.yaml
 
 .Verification
 
+ifdef::openshift-enterprise[]
 * Ensure that {VirtProductName} deployed successfully by watching the `PHASE` of the cluster service version (CSV) in the `openshift-cnv` namespace. Run the following command:
+endif::openshift-enterprise[]
+ifdef::openshift-origin[]
+* Ensure that {VirtProductName} deployed successfully by watching the `PHASE` of the cluster service version (CSV) in the `kubevirt-hyperconverged namespace`. Run the following command:
```

Comment:
"namespace" should be outside the backticks, like `openshift-cnv` namespace above. 

It might be simpler to create new attributes for the installation namespace and operator source. Then you wouldn't need all the ifdefs. Just a suggestion.

Reply:
Fixed; thanks